server/bin/fxa-content-server.js

```diff
@@ -151,6 +152,21 @@ function makeApp() {
   // The error handler must be before any other error middleware
   app.use(raven.ravenModule.middleware.express.errorHandler(raven.ravenMiddleware));
 
+  // log any joi validation errors
+  app.use((err, req, res, next) => {
```

Comment:
@vladikoff, @jrgm - I wonder if the errors are not logged because this comes after the raven middleware above.